### DISCUSSION TITLE
使用VcImageryProviderWmts服务时一直发送请求

### DISCUSSION THREAD

**Discussion by jiege15121313** [zouyaoji/vue-cesium#305] [General]
使用VcImageryProviderWmts加载WMTS服务时，浏览器一直在发送WMTS瓦片图像的请求（一直在发送请求，不会停止也不会间断），虽然这样地图也能加载出来，但是对运行速度影响很大，这是什么原因呢？
![d](https://user-images.githubusercontent.com/115357712/199186782-27f4a8d7-3248-4cf8-8751-5a81f008061e.png)

**Comment by zouyaoji** [comment]
如果没有改变相机的情况下，在线文档没有复现你说的问题哈

https://zouyaoji.top/vue-cesium/#/zh-CN/component/providers/vc-imagery-provider-wmts

如果相机一直在改变，它就会一直在请求，渲染机制就这样吧。 项目中我一般是将地图服务放与业务api不同的域名，没域名考虑用与主业务不同的端口。
